### DISCUSSION TITLE
Welcome to converge_cfd_user Discussions!

### DISCUSSION THREAD

**Discussion by sidbannet** [sidbannet/converge_cfd_user#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sidbannet** [comment]
@acrlakshman This is a great suggestion. Thanks and let me know if this discussion board looks okay.

**Reply by acrlakshman** [reply]
@sidbannet thank you for enabling this.
